converter/src/main/java/gov/cms/qpp/conversion/Converter.java

```diff
@@ -270,7 +270,8 @@ public void setReportDetails(AllErrors details) {
 		 * @return input stream of QPP validation details
 		 */
 		public InputStream streamRawValidationDetails() {
-			return new ByteArrayInputStream(qppValidationDetails.getBytes(Charset.defaultCharset()));
+			String raw = (qppValidationDetails != null) ? qppValidationDetails : "";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Convert this usage of the ternary operator to an "if"/"else" structure. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1774)